node/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM node:lts
-RUN npm install --global npm@8
+FROM node:17.6-slim
+RUN npm install --global npm@8.5.1
```

Comment:
For the record: we detected that node:lts has many vulnerabilities so we switched to this image instead.